phpunit.xml

```diff
@@ -1,6 +1,6 @@
 <phpunit
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/4.5/phpunit.xsd"
+    xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
```

Comment:
I don't believe this namespace change is correct. I don't see an HTTPS version for the xsi namespace in the spec: https://www.w3.org/TR/xmlschema-1/

Regardless, a namespace is not intended to be fetched.

See also https://github.com/sebastianbergmann/phpunit/blob/b25cb0dc1f77f4187f13c295784fec2b8b6060c6/phpunit.xml#L2

```suggestion
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Reply:
Fixed, rebased and repushed.

Reply:
I think this also applies to most namespaces used in CSS `@namespace` examples.

Reply:
Changed and repushed.